src/app/frontend/chrome/chrome.scss

```diff
@@ -22,6 +22,7 @@
 
 .kd-toolbar {
   box-shadow: $whiteframe-shadow-z1;
+  min-height: 64px;
```

Comment:
This is not a bug, this is a feature. One size is for desktops, other for mobiles in landscape mode and final for mobiles in portrait mode. See https://github.com/angular/material/blob/HEAD/src/components/toolbar/toolbar.scss

I think you can ignore this and have only one position for the FAB. Nobody is resizing the window like us, during development :) You can also copy the media queries and have the fab aligned in all views